receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -328,7 +328,7 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 				sum.SetIsMonotonic(true)
 				sum.SetAggregationTemporality(pmetric.AggregationTemporalityCumulative)
 			case writev2.Metadata_METRIC_TYPE_HISTOGRAM:
-				metric.SetEmptyHistogram()
```

Comment:
Unfortunately this is not that simple. Both histograms with explicit buckets (aka classic histograms) and histograms with exponential buckets share the same type ` writev2.Metadata_METRIC_TYPE_HISTOGRAM`, you need to decide which one it is by looking at the samples. If ts.Histograms is non nil, then you have an exponential histogram.

Reply:
@krajorama do you know if there is a documentation with these details? I think that would be helpful to keep it here, on the code. If it doesnt exists, I think that would be nice to just leave a comment

Reply:
Good point @perebaj about the documentation. There's [this](https://prometheus.io/docs/specs/native_histograms/) (and Carrie's document is linked on that page as well), but the [remote-write section](https://prometheus.io/docs/specs/native_histograms/#remote-write-read), and as far as I could understand, doesn't explain how to differentiate classic and native histograms in a remote-write v2 request.

Reply:
Yeah, we could definitely improve the documentation here

Reply:
resolved